src/gov-action-contracts/AIPs/AIP1Point1Target.sol

```diff
@@ -7,8 +7,8 @@ pragma solidity 0.8.16;
 /// and for bookkeeping.
 /// @dev note that this is not a "Gov Action" contract and thus does not conform to that standard.
 contract AIP1Point1Target {
-    address immutable public treasuryTimelock;
-    address immutable public arbitrumFoundationWallet;
+    address public immutable treasuryTimelock;
+    address public immutable arbitrumFoundationWallet;
```

Comment:
was this a linter change? the code was already deployed so changing it now will lead to code verification inconsistencies (ie someone checking deployed code vs repo code)

Reply:
updated